src/Data/DataInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace RonRademaker\Exporter\Data;
+
+use Iterator;
+
+/**
+ * Interface defining input data format
+ *
+ * @author Ron Rademaker
+ */
+interface DataInterface extends Iterator
+{
+    /**
+     * Gets the depth of the information in this DataInterface
+     *
+     * @return integer
+     */
+    public function getDepth();
```

Comment:
This will be used by the support function, for example a CSV can only handle data that has a depth of 1